### ISSUE TITLE
Editor Bug: Heights in Terrain. Not Working

### ISSUE THREAD

**Issue by GabrielMaestre** [SmartlyDressedGames/Unturned-3.x-Community#3110]
Hello how are you?
After the last update of unturned. I can't edit (Heights) on my map.
I was making this issue, like: editor ring not working.

But I tested it deep, it seems there is a small bug.
Every time I open the editor, even when restarting the game, the yellow ring does not appear, apparently either it does not update the appear function or it has some negative value (or zero). And so it doesn't appear unless you change the size,
same happens in: Terrain > Materials & Resources (red ring).

But the error is in the Heights, even changing the wisdom and it appearing again (yellow ring), it doesn't do its terrain functions. It just doesn't work.
(Raise, Lower, Smooth, Flatten).

![image](https://user-images.githubusercontent.com/25484986/161398627-3b5f180f-8a00-4d10-8f0d-7e8e542e8570.png)
![image](https://user-images.githubusercontent.com/25484986/161398633-482d1963-7533-481a-84f3-0caceb8047a3.png)

I tested, other functions like: Add and Remove: Items, Zombies, Nodes, Cars, Animals.
And they all work fine.

**Comment by SDGNelson** [comment]
It looks like brush Size and Strength are zero - if you increase does it work? I just double-checked with the Destruction map which is using old terrain and it worked properly.

**Comment by GabrielMaestre** [comment]
Yes I tested it, both without visibility (size 0) and increasing size.
I even tried using: Check File Integrity

**Comment by GabrielMaestre** [comment]
Hello again, I went deeper, and found that it's only 2 maps of mine that are like this (still strange, I didn't make any changes to them and when I opened them after the update I can't move the terrain anymore) I tested it with another map a little older and it worked.
Can you tell me why?

**Comment by SDGNelson** [comment]
To clarify: regardless of size, you need to increase strength above zero. If using a >0 size AND >0 strength does not work then if you can send your map file to nelson@smartlydressedgames.com please I will give it a try and see if I can replicate the problem.

**Comment by GabrielMaestre** [comment]
I'm not at home at the moment I'll do another test tomorrow as soon as it arrives if it doesn't work I'll send you an email, thank you.

**Comment by GabrielMaestre** [comment]
Hello again, I managed to solve the error by replacing some map files, files: Folder Editor. And deleting and adding again file: .unlocker
Note: I'm not the creator but I have permission to edit the map.
I'm not sure which of these things I did that made it work again.
[Feature Request] > Values in: Noise, Size, Strength. same as: Height (print example)
![image](https://user-images.githubusercontent.com/25484986/161640514-ce146a7c-2cb1-4f35-9dee-274b4e86896c.png)

But I have another bug to report.
Title: "Safezone in High places".

I saw very few people with the same bug, just one day I opened the editor and when I went to test the map every place on the map at a certain height was safezone (obs.: no safezone node added). There was no way I could solve it, I was forced to restore an old backup of the file: Nodes.dat

Sorry for take your time.
